doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -777,6 +777,10 @@ Other libraries
 
     * Fixed a bug where the random key would not be deleted from RAM after being generated and written.
 
+  * Flash Patch:
+
+    * Allow the :kconfig:option:`CONFIG_DISABLE_FLASH_PATCH` option to be used on nRF52833 devices.
```

Comment:
```suggestion
    * Allow the :kconfig:option:`CONFIG_DISABLE_FLASH_PATCH` Kconfig option to be used on the nRF52833 DK.
```

Reply:
Strictly speaking, the support is for the nRF52833 SOC, but if there's a convention to mention the DK, then no problem.

Reply:
Changed to

> Allow the :kconfig:option:`CONFIG_DISABLE_FLASH_PATCH` Kconfig option to be used on the nRF52833 SoC.

Tell me if I should revert to "DK"

Reply:
Better to keep SoC.